EverestSplash/EverestSplash.cs

```diff
@@ -268,15 +294,29 @@ public class EverestSplashWindow {
         
         // Okay, good code continues here
 
-        // FNA fixes
-        // FNA disables the cursor on game creation and when creating the window
-        fnaFixes.Add(
-            new FNAFixes.FNAFix(
-            () => SDL.SDL_ShowCursor(SDL.SDL_QUERY) == SDL.SDL_DISABLE,
-            () => SDL.SDL_ShowCursor(SDL.SDL_ENABLE),
-            () => SDL.SDL_ShowCursor(SDL.SDL_DISABLE)
-            )
-        );
+        if (randomness.WithChance(0.01)) {
+            string[] possibleTexts = {
+                "Adding %s to the pie",
+                "Asking Madeline for %s",
+                // "Fighting your inner %s", 
+                // "Adding golden berries to %s",
+                "Looking for %s",
```

Comment:
Confusing, suggests that Everest is having issues locating files.

Reply:
Fixed